lib/stimulus_reflex/configuration.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module StimulusReflex
+  class << self
+    def configure
+      yield configuration
+    end
+
+    def configuration
+      @configuration ||= Configuration.new
+    end
+
+    alias config configuration
+  end
+
+  class Configuration
+    attr_accessor :example # put your attr_accessors here, like `attr_accessor: debug`, remove this line when you do.
+
+    def initialize
```

Comment:
```suggestion
    # Assign a default to a configurable variable
    # def initialize(example)
    #     @example = true
    # end
```

Reply:
dito, but I'll do it, it is a bit cleaner not having an actual dummy config var implemented. Though on second thought, without an implemented variable, you cannot test this out with your app.

Reply:
The problem is: you have nothing to be configured/set up yet. This PR is a groundwork for future configurable variables like `debug`, `logging` etc.

Reply:
I understand that. My point was, if somebody wants to check out this branch, they can test it with the example config variable. But that point will be moot, once this PR gets merged. 

I mean basically it shouldn't matter anyway, as there should be no release with this PR merged, if we don't also merge one of the PRs that actually needs this :-)